src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/FeedbackReviews/FeedbackReviews.tsx

```diff
@@ -152,18 +89,17 @@ export const FeedbackReviews = () => {
           setAssessmentStep={setAssessmentStep}
           submitCallback={submitAssessmentCallback}
           setIsLoading={setSubmitAssessmentLoading}
-          answerId={answerId}
           setError={setSubmitAssessmentError}
           error={submitAssessmentError}
           userName={userName}
         />
       )}
       <Reviews
         isLoading={isLoading}
-        reviewsError={reviewsError ? getErrorMessage(reviewsError) : null}
-        reviewersData={reviewersData}
-        removeReviewError={removeReviewError ? getErrorMessage(removeReviewError) : null}
-        removeReviewLoading={removeReviewLoading}
+        reviewError={reviewsError || reviewsFlowsError}
+        reviewerData={reviewerData}
```

Comment:
The same here:
Maybe it's better to keep reviewers plural since it represents data for multiple reviewers, not just a single reviewer?